src/pywrangler/sync.py

```diff
@@ -111,6 +126,10 @@ def create_pyodide_venv():
         )
         return
 
+    # Workaround to fix caching issue on some machines.
+    logger.debug("Installing xbuildenv...")
+    run_command([str(pyodide_cli_path), "xbuildenv", "install"])
```

Comment:
Add a link to the PR fixing it:
https://github.com/pyodide/pyodide-build/pull/239